mslib/msui/msui_mainwindow.py

```diff
@@ -671,6 +671,8 @@ def handle_import_local(self, extension, function, pickertype):
             self.last_save_directory,
             f"Flight Track (*.{extension});;All files (*.*)",
             pickertype=pickertype)
+        if filenames is None:
+            raise TypeError("'NoneType' object is not iterable")
```

Comment:
a raise is not needed, the user decided to cancel his request. The user knows already that it was canceled.

Try `if filenames is not None:`